libs/entry-components/dialog/dialogs/entry-dialog.component.html

```diff
@@ -1,33 +1,19 @@
-<div class="entry-dialog dialog-form">
-    <div class="dialog-header">
-        <h1 class="title">{{ title }}</h1>
+<h1 mat-dialog-title>{{ title }}
+    <button mat-icon-button mat-dialog-close aria-label="Close dialog" *ngIf="!hideClose">
```

Comment:
We still need to prettify this, cover edge cases, i.e. small dialog, long title, adequate button positioning etc.
Make a list and give it to Svetlana.